samples/helloworldasync/helloworld.csproj

```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+	<LangVersion>7.1</LangVersion>
```

Comment:
The 2.0 SDK support LangVersion 7.1 by default. I don't think you need this line at all.

Reply:
It is necessary, because the default `LangVersion` is `default`, which is the latest major version.

Reply:
@jaredpar

Reply:
My goal is for samples to be as simple as possible. (correct as well of course).